pep/tests/integration/helpers/embargo-year-test.ts

```diff
@@ -0,0 +1,19 @@
+import { render } from '@ember/test-helpers';
+
+import { setupRenderingTest } from 'ember-qunit';
+
+import hbs from 'htmlbars-inline-precompile';
+import { module, test } from 'qunit';
+
+module('Integration | Helper | embargo-year', function(hooks) {
+    setupRenderingTest(hooks);
+
+    // Replace this with your real tests.
+    test('it renders', async function(assert) {
+        this.set('inputValue', '1234');
+
+        await render(hbs`{{embargo-year inputValue}}`);
+
+        assert.equal(this.element.textContent?.trim(), '1234');
```

Comment:
doesnt look like that would pass based on what the helper is doing, i dont _think_.